dir.traversal.targets

```diff
@@ -197,6 +197,10 @@
 
   <Target Name="GetFilesToPackage" DependsOnTargets="FilterProjects"
           Returns="@(FilesToPackage)">
+    <PropertyGroup>
+      <BuildInParallel Condition="'$(BuildInParallel)' == '' AND '$(MSBuildNodeCount)' > '1'">true</BuildInParallel>
```

Comment:
> AND '$(MSBuildNodeCount)' > '1' [](start = 60, length = 32)

this won't do anything, if node count is 1 then it will just ignore build in parallel